src/bandersnatch/tests/test_verify.py

```diff
@@ -196,12 +207,11 @@ async def test_delete_unowned_files() -> None:
 
 @pytest.mark.asyncio
 async def test_get_latest_json(monkeypatch: MonkeyPatch) -> None:
-    config = FakeConfig()
     executor = ThreadPoolExecutor(max_workers=2)
     json_path = Path(gettempdir()) / f"unittest_{os.getpid()}.json"
     master = Master("https://unittest.org")
-    master.url_fetch = do_nothing  # type: ignore
-    await get_latest_json(master, json_path, config, executor)  # type: ignore
+    master.url_fetch = fake_fetch  # type: ignore
+    await get_latest_json(master, json_path, executor)
```

Comment:
Weird - We don't assert anything here :| - Should we check a json file exists or something?

Reply:
I'll look into that, I didnt notice that as I was just doing some refactoring , assuming this is just fine.

Reply:
A tiny assert is added, and it passed! This test if for `get_latest_json` function, which is to fetch json from pypi.org, this test patched url_fetch method and return a fake text , this test assert the fake text is written to json file